packages/webapp/src/containers/Finances/ActualRevenue/index.jsx

```diff
@@ -30,8 +32,8 @@ export default function ActualRevenue({ history, match }) {
     mode: 'onBlur',
     shouldUnregister: true,
     defaultValues: {
-      from_date: moment().startOf('year').format('YYYY-MM-DD'),
-      to_date: moment().endOf('year').format('YYYY-MM-DD'),
+      from_date: new Date(dateRange.startDate).toISOString().split('T')[0],
```

Comment:
I believe that `dateRange.startDate` is equivalent to `new Date(dateRange.startDate).toISOString().split('T')[0]`. 
Same comment applies elsewhere in this commit.

Reply:
Good point, I've resolved this now